apps/condo/domains/user/utils/testSchema/index.js

```diff
@@ -92,8 +92,9 @@ async function registerNewUser (client, extraAttrs = {}, { raw = false } = {}) {
     })
     if (raw) return { data, errors }
 
-
-    // expect(errors).toEqual(undefined)
+    if (typeof expect === 'function') {
```

Comment:
xD nice %)

Reply:
Could you please mention it in docs file?